x/smart-account/ante/ante_test.go

```diff
@@ -341,29 +337,20 @@ func (s *AuthenticatorAnteSuite) TestSpecificAuthenticator() {
 			)
 
 			anteHandler := sdk.ChainAnteDecorators(s.AuthenticatorDecorator)
-			res, err := anteHandler(s.Ctx.WithGasMeter(sdk.NewGasMeter(300000)), tx, false)
+			_, err := anteHandler(s.Ctx.WithGasMeter(storetypes.NewGasMeter(300000)), tx, false)
 
 			if tc.shouldPass {
 				s.Require().NoError(err, "Expected to pass but got error")
 			} else {
 				s.Require().Error(err, "Expected to fail but got no error")
 			}
-
-			// ensure only the right amount of sigs have been checked
-			if tc.checks > 0 {
-				s.Require().GreaterOrEqual(res.GasMeter().GasConsumed(), uint64(baseGas+(tc.checks-1)*approachingGasPerSig))
-				s.Require().LessOrEqual(res.GasMeter().GasConsumed(), uint64(baseGas+tc.checks*approachingGasPerSig))
-			} else {
-				if tc.checkGas {
-					s.Require().LessOrEqual(res.GasMeter().GasConsumed(), uint64(baseGas))
-				}
-			}
```

Comment:
Note to self: come back to this

I don't quite get why this is removed

Reply:
Actually this was likely removed by @nicolaslara when he corrected the remaining issues in the smart accounts module. He will likely be able to expand further on why this was safe (good call pointing it out though)

Reply:
I think we can look at this module by itself after the merge